sdk/servicenetworking/arm-servicenetworking/package.json

```diff
@@ -1,113 +1,121 @@
+      }
+    ]
+  },
+  "dependencies": {
+    "@azure-rest/core-client": "^2.3.1",
+    "@azure/abort-controller": "^2.1.2",
+    "@azure/core-auth": "^1.9.0",
+    "@azure/core-lro": "^3.1.0",
+    "@azure/core-rest-pipeline": "^1.18.2",
+    "@azure/core-util": "^1.11.0",
+    "@azure/logger": "^1.1.4",
+    "tslib": "^2.8.1"
+  },
   "devDependencies": {
-    "@azure-tools/test-credential": "^1.0.0",
-    "@azure-tools/test-recorder": "^3.0.0",
+    "@azure-tools/test-credential": "^2.0.0",
+    "@azure-tools/test-recorder": "^4.1.0",
+    "@azure-tools/test-utils-vitest": "^1.0.0",
     "@azure/dev-tool": "^1.0.0",
-    "@azure/identity": "^4.0.1",
-    "@types/chai": "^4.2.8",
-    "@types/mocha": "^10.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@azure/identity": "^4.6.0",
+    "@microsoft/api-extractor": "^7.40.3",
     "@types/node": "^18.0.0",
-    "chai": "^4.2.0",
+    "@vitest/browser": "^3.0.3",
+    "@vitest/coverage-istanbul": "^3.0.3",
     "dotenv": "^16.0.0",
-    "mocha": "^11.0.2",
-    "ts-node": "^10.0.0",
-    "typescript": "~5.7.2"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Azure/azure-sdk-for-js.git"
-  },
-  "bugs": {
-    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+    "eslint": "^9.9.0",
+    "playwright": "^1.49.1",
+    "typescript": "~5.6.2",
+    "vitest": "^3.0.3"
   },
-  "files": [
-    "dist/**/*.js",
-    "dist/**/*.js.map",
-    "dist/**/*.d.ts",
-    "dist/**/*.d.ts.map",
-    "dist-esm/**/*.js",
-    "dist-esm/**/*.js.map",
-    "dist-esm/**/*.d.ts",
-    "dist-esm/**/*.d.ts.map",
-    "src/**/*.ts",
-    "README.md",
-    "LICENSE",
-    "tsconfig.json",
-    "review/*",
-    "CHANGELOG.md",
-    "types/*"
-  ],
   "scripts": {
-    "build": "npm run clean && tsc && dev-tool run bundle && npm run minify && npm run extract-api",
-    "build:browser": "echo skipped",
-    "build:node": "echo skipped",
-    "build:samples": "echo skipped.",
-    "build:test": "echo skipped",
-    "check-format": "echo skipped",
+    "build": "npm run clean && dev-tool run build-package && dev-tool run extract-api",
+    "build:samples": "dev-tool run typecheck --paths samples-dev/*.ts && dev-tool samples publish -f",
+    "build:test": "npm run clean && dev-tool run build-package && dev-tool run build-test",
+    "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.{ts,cts,mts}\" \"test/**/*.{ts,cts,mts}\" \"*.{js,cjs,mjs,json}\" \"samples-dev/*.ts\"",
     "clean": "dev-tool run vendored rimraf --glob dist dist-browser dist-esm test-dist temp types *.tgz *.log",
-    "execute:samples": "echo skipped",
-    "extract-api": "dev-tool run extract-api",
-    "format": "echo skipped",
+    "execute:samples": "dev-tool samples run samples-dev",
+    "extract-api": "dev-tool run vendored rimraf review && dev-tool run vendored mkdirp ./review && dev-tool run extract-api",
```

Comment:
No calls to `mkdirp` should be required anymore as extract-api now has it built-in